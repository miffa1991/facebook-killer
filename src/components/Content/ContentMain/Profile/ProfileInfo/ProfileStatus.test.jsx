import React from "react";
import { create } from "react-test-renderer";
import PropfileStatusContainer from "./ProfileStatus";

describe("ProfileStatus component", () => {
	test("status from props should be state", () => {
		const component = create(<PropfileStatusContainer />);
		const instance = component.getInstance();
		expect(instance.state.status).toBe('-----');
	});
});