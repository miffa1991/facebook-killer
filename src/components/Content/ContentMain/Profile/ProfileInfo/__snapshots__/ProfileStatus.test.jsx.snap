// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileStatus component status from props should be state 1`] = `
<div>
  <div>
    <span
      onClick={[Function]}
    >
      -----
    </span>
  </div>
</div>
`;
